readme.md

```diff
@@ -0,0 +1,30 @@
+Earl is a Discord bot designed to try to be your friend, but also to poke your buttons and be a bit provacative.
+He responds and reacts to certain phrases, words, and the like, to try to help uplift and motivate your community.
```

Comment:
Imagine the child of Dogbert and Veggietales Facts being described as uplifting.